src/main/resources/templates/lemminx/clientSettings.json

```diff
@@ -6,6 +6,9 @@
     "useContextAwareSorting": true,
     "useTemplateForInvocationOnlySnippet": false
   },
+  "editor": {
```

Comment:
Disabled by default for XML. Note that all JetBrains IDEs already have breadcrumbs support for XML anyway.